streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -239,7 +246,7 @@ public void run() {
         } catch (Exception e) {
             // we have caught all Kafka related exceptions, and other runtime exceptions
             // should be due to user application errors
-            log.error(String.format("stream-thread [%s] Streams application error during processing: ", this.getName()),  e);
+            log.error(logPrefix + " Streams application error during processing: ",  e);
```

Comment:
Yeah, either way works for me. Seems unlikely someone using a bleeding edge library like streams would be on an ancient Slf4j 😉